zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/RecordIndexRouter.java

```diff
@@ -86,6 +86,6 @@ String routingFor(final Record<?> record) {
   }
 
   private String valueTypeToString(final ValueType valueType) {
-    return valueType.name().toLowerCase().replace("_", "-");
+    return valueType != null ? valueType.name().toLowerCase().replace("_", "-") : "";
```

Comment:
❓ Why is this neccessary?

Reply:
I'm not sure on the context of this change so can't speak for why its here, I would imagine that by this point we always have the value type so it shouldn't be null?

Reply:
Exactly, ValueType is never null here